torchvision/transforms.py

```diff
@@ -516,6 +516,39 @@ def adjust_gamma(img, gamma, gain=1):
     return img
 
 
+def rotate(img, angle, resample=False, expand=False, center=None, translate=None):
```

Comment:
I think it would be better to split this into two transforms. i.e. one for rotation and one for translation, the user can then chain them together if they want both?

Reply:
Hi @alykhantejani thanks for the coments :)

I also think that translation should have it's own function, but I just keeped the translate keyword argument to be faithful to the original `PIL.Image.rotate` API. 

Do you think that it should be better to remove the translate argument?

Reply:
yeah I personally would prefer a separate rotate and translate transform, and removing the translate param to rotate

Reply:
Removed the translate arg and adapted to the new functional/transform API. I'll probably work in another pull request to support affine transforms using  PIL.Image.transform